Review.java

```diff
@@ -165,4 +165,30 @@ public static String randomAdjective()
       return randomNegativeAdj();
     }
   }
+  
+  public static double totalSentiment(String fileName)
+  {
+    String file = textToString(fileName);
+    String word = "";
+    String space = " ";
+    double totalVal = 0.0;
+    for(int i = 0; i < file.length(); i++)
+      {
+        String letter = file.substring(i, i + 1);
+    if(letter.equals(space))
+      {
+        removePunctuation(word);
+        totalVal += sentimentVal(word);
+        word = "";
+      }
+    else
+{
+word += letter;
+}
+}
+removePunctuation(word);
+totalVal += sentimentVal(word);
+return totalVal;

```

Comment:
Everything looks great! My only suggestion is indenting the rest of your code so it lines up with the other code. Other than that looking good!

Reply:
Thank you Malaya I have just fixed the Indentation!